src/components/StarRating/src/StarRating.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+	<div :style="starComputedStyles">
+		<star
+			v-for="star in stars"
+			:key="star"
+			:fill="fillForStar(star)"
+			:class="$s.Star"
+			@hover="hoverStar(star)"
+			@unhover="unhoverStar(star)"
+			@click="clickStar(star)"
+		/>
+	</div>
+</template>
+
+<script>
+import Star from './Star.vue';
+
+const MIN_RATING = 0;
+const MAX_RATING = 5;
+const HALF_RATING = 0.5;
+// eslint-disable-next-line no-magic-numbers
+const stars = [1, 2, 3, 4, 5];
```

Comment:
Instead of creating an array, we can create a number const: `STAR_COUNT = 5`.

In the template we can loop `v-for="star in STAR_COUNT"`